### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by MrBourriz** [geumjin99/PyRINEX#1]
There's any documentation coming soon ?

**Comment by geumjin99** [comment]
Yes, it's part of an academic paper, and a detailed description of it will be available after publication.And I'm thinking about writing a manual after my vacation, about two weeks from now.
